tests/Unit/Helper/MiscHelperTest.php

```diff
@@ -33,4 +33,12 @@ public function getInputOutputForSplitSecondsMicroseconds()
             array('3.123456', array(3, 123456)),
         );
     }
+
+    public function testHexDump()
+    {
+        $res = MiscHelper::hexdump('FM', 'hmtlOutput' == false, 'uppercase' == false, 'return' == true);
```

Comment:
I'm not a maintainer, just saying what I believe is the best. Take as much time as you need, @videlalvaro is probably also going to take some time :) 

And thank you for fixing this!